app/src/pages/outside/registrationPage/registrationForm/registrationForm.jsx

```diff
@@ -84,20 +84,30 @@ export class RegistrationForm extends Component {
     autofill: PropTypes.func.isRequired,
     email: PropTypes.string,
     loading: PropTypes.bool,
+    initData: PropTypes.object,
   };
 
   static defaultProps = {
     submitForm: () => {},
     email: '',
     loading: false,
+    initData: {},
   };
 
   componentDidMount = () => {
-    this.autofillEmail();
+    this.autofillData();
   };
 
   autofillEmail = () => this.props.autofill('email', this.props.email);
 
+  autofillData = () => {
+    this.autofillEmail();
+    this.props.autofill('login', this.props.initData.login);
+    this.props.autofill('name', this.props.initData.fullName);
+    this.props.autofill('password', this.props.initData.password);
+    this.props.autofill('confirmPassword', this.props.initData.password);
```

Comment:
As these data initializes `reduxForm`, they can be provided via [`initialValues`](https://redux-form.com/8.3.0/docs/api/reduxform.md/#-code-initialvalues-object-lt-string-string-gt-code-optional-) prop, so it will be set automatically.

Reply:
Yes, I've tried, but in fact that some field has different names it's not working for all fields